tst/RequestTest.php

```diff
@@ -54,7 +42,25 @@ public function testView()
     public function testRead()
     {
         $this->reset();
-        $id                        = $this->getRandomId();
+        $id                        = Helper::getRandomId();
+        $_SERVER['REQUEST_METHOD'] = 'GET';
+        $_SERVER['QUERY_STRING']   = $id;
+        $_GET[$id]                 = '';
+        $request                   = new Request;
+        $this->assertFalse($request->isJsonApiCall(), 'is HTML call');
+        $this->assertEquals($id, $request->getParam('pasteid'));
+        $this->assertEquals('read', $request->getOperation());
+    }
+
+    /**
+     * paste IDs are 8 bytes hex encoded strings, if unlucky, this turns into
+     * a numeric string that PHP will cast to an int, for example in array keys
+     * @see https://www.php.net/manual/en/language.types.array.php
```

Comment:
```suggestion
     * @see https://www.php.net/manual/en/language.types.array.php
     * @see https://github.com/PrivateBin/PrivateBin/issues/1435
```

(optional) Maybe we could link the issue, too, but any way is fine for me. It's explained, that's what matters…

Reply:
The issue should already be linked in the git comment. IMO git comments is where you track the why a code got changed and comments in the code are for explaining the purpose and intent of a piece of code if it is non-obvious from just looking at the code.